### ISSUE TITLE
UK Thermostat - eco_temperature_low_c is undefined

### ISSUE THREAD

**Issue by mriksman** [chrisjshull/homebridge-nest#22]
I have a Heat-only Nest (UK Nest - we don't have AUTO or COOLING). 

My Nests (for the example below) are;
Guest: OFF. Target temp previously at 18 deg C. I have no idea where it is getting 20 deg C from for the target_temperature_low_c variable. When it is ON, it displays 18 correctly.
Living: ECO. Eco temperature is set to 18 deg C. In Home app it shows 10. Looking at the variables it uses to obtain the TargetTemperature - there is clearly something wrong;

```
[3/9/2018, 6:17:36 AM] [Nest] Guest Thermostat target_temperature_c: 18
[3/9/2018, 6:17:36 AM] [Nest] Guest Thermostat target_temperature_low_c: 20
[3/9/2018, 6:17:36 AM] [Nest] Living Room Thermostat eco_temperature_low_c: undefined
[3/9/2018, 6:17:36 AM] [Nest] Living Room Thermostat eco_temperature_low_c: undefined
```

When I set Guest to ECO, I get the same;
```
[3/9/2018, 6:17:52 AM] [Nest] Guest Thermostat eco_temperature_low_c: undefined
[3/9/2018, 6:17:52 AM] [Nest] Guest Thermostat eco_temperature_low_c: undefined
```

Log entries located at lines 241;
```
                case "eco":
                        this.log(this.name + "eco_temperature_low_c: " + this.device.eco_temperature_low_c);
                        return this.device.eco_temperature_low_c;
                case "heat-cool":
                default:
                        this.log(this.name + "target_temperature_low_c: " + this.device.target_temperature_low_c);
                        return this.device.target_temperature_low_c;
```

And here at line 226 ;

```
                default:
                        this.log(this.name + "target_temperature_c: " + this.device.target_temperature_c);
                        return this.device.target_temperature_c;
```

**Comment by mriksman** [comment]
I have confirmed, at least for the UK Nest, that eco_temperature does not exist. When in AWAY/ECO, the correct temperature is available in **away**_temperature_low_c

#21 will return away_temperature if eco_temperature is undefined.

**Comment by chrisjshull** [comment]
fix pushed as v2.1.3!
